ansible/roles/sidekiq/tasks/main.yml

```diff
@@ -0,0 +1,44 @@
+---
+  - name: Install monit
+    apt:
+      name: monit
+      state: present
+    sudo: true
+
+  - name: Install sidekiq dependencies
+    apt:
+      name: "{{item}}"
+      state: present
+    with_items:
+      - redis-server
+    sudo: true
+
+  - name: Install sidekiq monit script
+    template:
+      src: sidekiq-monit-rc
+      dest: /etc/monit/conf.d/sidekiq_{{project_name}}_{{rails_env}}
+    sudo: true
+    notify:
+      - monit stop all
+      - monit reload
+      - monit start all
```

Comment:
we should combine this to `notify: reload monit` or something simple and have that handler do all three if possible.